core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.ts

```diff
@@ -487,6 +497,9 @@ angular.module('oppia').component('learnerDashboardPage', {
 
         $q.all([userInfoPromise, dashboardDataPromise]).then(function() {
           LoaderService.hideLoadingScreen();
+          setTimeout(() => {
+            ctrl.addFocusWithoutScroll('ourLessonsBtn');
+          }, 0);
```

Comment:
Does this work all the time? Let's add a comment. Ditto everywhere you have introduced setTimeout for this purpose.

Reply:
done. Yes it works almost everytime